autoload/LanguageClient.vim

```diff
@@ -27,9 +27,31 @@ function! s:EchoEllipsis(message) abort
     echo s:Ellipsis(a:message)
 endfunction
 
+" Credit: ALE, snippets from ale#cursor#TruncatedEcho()
+function! s:Truncate(message) abort
```

Comment:
Please update this function name or merge the function body into EchomsgEllipsis.

Reply:
Done.